src/main/kotlin/eu/europa/ec/eudi/pidissuer/domain/CNonce.kt

```diff
@@ -19,7 +19,6 @@ import java.time.Duration
 import java.time.Instant
 
 data class CNonce(
-    val accessToken: String,
     val nonce: String,
     val activatedAt: Instant,
     val expiresIn: Duration,
```

Comment:
I don't the CNonce needs to have "shape".

CNonce = Nonce

Only two classes should be aware of the structure of the c_nonce
Generator / Validator.